### DISCUSSION TITLE
spacy

### DISCUSSION THREAD

**Discussion by TheCodeCache** [flet-dev/flet#3248] [Packages] [answered]
https://pypi.org/project/spacy/

**Comment by NHLOCAL** [comment]
I agree with the request. Has the package been added?

**Comment by NHLOCAL** [comment]
All non-Pure Python dependencies for spacy:

cymem
MarkupSafe
marisa-trie
murmurhash
numpy
preshed
pydantic-core
charset-normalizer
srsly
thinc
blis
wrapt

**Comment by ndonkoHenri** [comment, marked as answer]
spacy 3.8.13 and its native dependency chain (cymem, murmurhash, preshed, srsly, thinc) are now available for Android and iOS: https://pypi.flet.dev/spacy
